less/input-groups.less

```diff
@@ -126,6 +126,15 @@
   vertical-align: middle; // Match the inputs
 }
 
+.input-group-currency-select-btn {
+  min-width: 144px;
+}
+@media (max-width: @screen-xs-max) {
```

Comment:
Could we do the media query mobile first instead?

Reply:
As in having media query before line 129?

Reply:
Nah, as in having mobile as the default behaviour and using `min-width` media query to change appearance for wider viewports.